package.json

```diff
@@ -18,39 +18,21 @@
     "lint": "eslint . --ignore-path .gitignore",
     "lint:ci": "npm run lint -- -o lint-results.json -f json",
     "danger": "./bin/duti",
-    "duti:fix": "npm run lint -- --fix && npm run fmt",
-    "fmt": "prettier --write {*.js,{src,bin}/**/*.js} --no-semi --single-quote --trailing-comma es5"
-  },
-  "prettier": {
-    "singleQuote": true,
-    "semi": false,
-    "trailingComma": "es5"
-  },
-  "eslintConfig": {
-    "extends": "smartprocure",
-    "parserOptions": {
-      "ecmaVersion": 8,
-      "sourceType": "module"
-    },
-    "env": {
-      "node": true
-    }
+    "duti:fix": "npm run lint -- --fix"
   },
   "dependencies": {
     "bluebird": "^3.5.0",
     "cosmiconfig": "^2.2.2",
-    "futil": "^1.54.0",
-    "lodash": "^4.17.11",
+    "futil": "1.69.0",
+    "lodash": "4.17.21",
     "pkg-up": "^2.0.0",
     "strip-ansi": "^4.0.0",
     "underscore.string": "^3.3.4"
   },
   "devDependencies": {
-    "danger": "^6.1.13",
-    "eslint": "^4.12.1",
-    "eslint-config-smartprocure": "^1.2.0",
-    "jest": "^20.0.4",
-    "prettier": "^1.5.3"
```

Comment:
Prettier is useful for text editing

Reply:
We need to minimize formatting changes in PRs though (e.g. you merge a PR that formats using eslint and I merge a PR that uses prettier).

We should add a format step on the `pull-request` action like we do in spark.

Reply:
No one should be formatting files with prettier unless it's specified in package.json.
What happens when developers use a global install of prettier and the rules don't match?

I removed the fmt script because it kept undoing the eslint-config-smartprocure rules.